tutorials/tutorial-livestream/src/main/kotlin/io/getstream/video/android/tutorial/livestream/LiveGuest.kt

```diff
@@ -63,6 +63,7 @@ fun LiveAudience(
         callServiceConfig = livestreamGuestCallServiceConfig(),
         ensureSingleInstance = false,
         loggingLevel = LoggingLevel(priority = Priority.VERBOSE),
+        audioUsage = AudioAttributes.USAGE_MEDIA,
```

Comment:
Isn't it better to just set this to `audioUsage = callServiceConfig.audioUsage` and deprecate the `audioUsage` parameter in the builder?
We always take it from the config for the peer connection as well.